src/crimson/os/seastore/cache.h

```diff
@@ -1281,7 +1284,7 @@ class Cache {
   {
     if (p_src && is_background_transaction(*p_src))
       return;
-    if (ext.is_clean() && !ext.is_placeholder()) {
```

Comment:
Looks all the use cases of `is_clean()` in `Cache.h/cc` can be replaced by the more accurate `is_stable_clean()`, right?

Reply:
yes

Reply:
done